### PR TITLE
chore: rename as per PEP 8

### PR DESCRIPTION
**Pull request by cpaniaguam**
General module renaming. A few changed files but straightforward to review.

### CODE REVIEW

File: pyproject.toml
Comment (hollandjg) [inline pyproject.toml:1]: The project name on line 13 needs updating here too.

Comment (hollandjg) [reply pyproject.toml:1]: We should also update everywhere where we list python versions – we're probably only compatible with 3.10 upwards now.

**Review by hollandjg — CHANGES_REQUESTED** [review]
Looks good! Just a couple of changes in the pyproject.toml file

File: pyproject.toml [hunk 1]
```diff
@@ -159,22 +158,22 @@ test = [
 # maintainers, and where to support the project financially. The key is
 # what's used to render the link text on PyPI.
 [project.urls]  # Optional
-"Homepage" = "https://github.com/brown-ccv/icesat2-tracks"
-"Bug Reports" = "https://github.com/brown-ccv/icesat2-tracks/issues"
-"Source" = "https://github.com/brown-ccv/icesat2-tracks"
```
Comment (hollandjg) [inline pyproject.toml:164]: We'll need to rename the repo to make this consistent. New issue for that?

**Review by hollandjg — APPROVED** [review]
Looks good to me!

File: pyproject.toml (hunk 1 above)
Comment (cpaniaguam) [reply pyproject.toml:164]: Sure, I can take care of that!

Comment (cpaniaguam) [reply pyproject.toml:164]: #190

**Pull request merged**